styles/component-styles/alert.css

```diff
@@ -0,0 +1,34 @@
+.alert-container{
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+}
+
+.error{
```

Comment:
The generic names like error, warning, success, info, etc. can cause confusion while maintaining the CSS. You can give a prefix, preferably the component these variations belongs to. In this case, the alert component

Reply:
Good thinking Shashi. 
Added alert as the prefix in all 4 names.

Thanks